src/components/ComboButton/ComboButton.stories.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file Combo button stories.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import centered from '@storybook/addon-centered/react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { withA11y } from '@storybook/addon-a11y';
+import { radios } from '@storybook/addon-knobs';
+import { Filter20 } from '@carbon/icons-react';
+
+import { patterns } from '../../../.storybook';
+
+import { TooltipDirection } from '../IconButton/IconButton';
+import { ComboButton } from '../..';
+
+import actions from './_mocks_';
+
+const props = () => ({
+  overflowMenuDirection: radios(
+    'Direction',
+    {
+      top: TooltipDirection.TOP,
+      bottom: TooltipDirection.BOTTOM,
+    },
+    TooltipDirection.TOP
+  ),
+});
+storiesOf(patterns('ComboButton'), module)
```

Comment:
What's the difference between a pattern and a component?

Reply:
A pattern is like a "complex component" that is made up of several pieces. And then a component is more like something core (and usually proxied from Carbon).

@petervachon did you have anything extra to add to this explanation? ☝️